From af7e11de69690e465e30632e546771ee3c3bdce7 Mon Sep 17 00:00:00 2001
From: Damien Lespiau <damien.lespiau@intel.com>
Date: Mon, 14 Jan 2013 23:25:14 +0000
Subject: build: Add CAIRO_FLAGS to the debugger compilation

The library in lib/ exposes <cairo.h> in its main header and thus users
must be able to include it.
---
diff --git a/debugger/Makefile.am b/debugger/Makefile.am
index d76e2ac..f1e49b9 100644
--- a/debugger/Makefile.am
+++ b/debugger/Makefile.am
@@ -11,6 +11,7 @@ AM_CPPFLAGS = 			\
 AM_CFLAGS = 			\
 	$(DRM_CFLAGS) 		\
 	$(PCIACCESS_CFLAGS) 	\
+	$(CAIRO_CFLAGS)		\
 	$(CWARNFLAGS)
 
 LDADD = $(top_builddir)/lib/libintel_tools.la $(DRM_LIBS) $(PCIACCESS_LIBS) $(CAIRO_LIBS)
--
cgit v0.9.0.2-2-gbebe
